cordova-react/merges/android/scripts/platformOverrides.js

```diff
@@ -0,0 +1,10 @@
+(function () {
```

Comment:
I think this is boiler plate that gets included in every cordova project. Probably just easier not to touch it. Rido, do users ever edit these?